README.md

```diff
@@ -1,2 +1,164 @@
-# spec-dfn-contract
-Contract for marking up definitions in specifications
+# Specification definitions contract
+
+This document defines the "specification definitions contract", known herein as, "the contract".
+
+The contract standardizes how an HTML definition (i.e., anything inside a `<dfn>` or `h1`-`h6` element) and respective attributes and values.
+
+Specifications that conform to the contract are, with the help of reference databases, able to unambiguously cross-reference each other's definitions.
+
+## Some examples
+
+A simple example of a definition:
+
+```HTML
+<dfn id="simple-def" data-export="" data-dfn-type="dfn">
+  I'm a simple definition
+</dfn>
+```
+
+A more complex examples:
+
+```HTML
+<!-- Example of defining an abstract operation -->
+<dfn
+  id="string-parser"
+  data-export=""
+  data-lt="parse a string"
+  data-dfn-type="abstract-op">
+  string parser
+</dfn>
+
+<!-- Example of defining a WebIDL interface -->
+<dfn
+  data-export=""
+  data-dfn-type="interface"
+  id="dom-paymentrequest"
+  data-dfn-for="">
+  <code>PaymentRequest</code>
+</dfn>
+
+```
+
+## Conformance
+
+This document serves as the canonical source for how definitions are marked-up.
+
+Conforming applications to this specification are primarily authoring tools that assist in generating technical specifications. However, this specification is also here to help people who wish to mark up documents manually (or simply want to understand why various attributes mean and what they do!).
```

Comment:
I think we have both producers (authoring tools here) and consumers (e.g. reffy, shepherd) that need to conform to the contract.

Reply:
```suggestion
Conforming applications to this specification are primarily authoring tools that assist in generating technical specifications. However, this specification is also here to help people who wish to mark up documents manually (or simply want to understand what various attributes mean and what they do!).
```